test/integration/targets/ec2_group/tasks/main.yml

```diff
@@ -1004,6 +1366,47 @@
           - 'not result.changed'
       when: result.ip_permissions_egress[0].ip_ranges[0].description is undefined
 
+    # ============================================================
+    - name: test removing rule descriptions (expected changed=true)
+      ec2_group:
+        name: '{{ec2_group_name}}'
+        description: '{{ec2_group_description}}'
+        <<: *aws_connection_info
+        vpc_id: '{{ vpc_result.vpc.id }}'
+        purge_rules_egress: false
+        purge_rules: false
+        state: present
+        rules:
+        - proto: "tcp"
+          ports:
+          - 8281
+          cidr_ipv6: 1001:d00::/24
+          rule_desc:
+        rules_egress:
+        - proto: "tcp"
+          ports:
+          - 8282
+          cidr_ip: 2.2.2.2/32
+          rule_desc:
+      register: result
+      ignore_errors: true
+
+    - name: assert that rule descriptions were removed (expected changed=true with newer botocore)
+      # Only assert this if rule description is defined as the botocore version may < 1.7.2.
+      # It's still helpful to have these tests run on older versions since it verifies backwards
+      # compatibility with this feature.
+      assert:
+        that:
+          - 'result.ip_permissions[0].ipv6_ranges[0].description is undefined'
```

Comment:
I need to run tests using an older version of botocore... on 2.5 this fails for some reason.